src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -149,15 +149,24 @@ public function countItems()
     /**
      * {@inheritdoc}
      */
-    public function isProductInTheList($productName)
+    public function isProductInTheList(string $productName): bool
     {
         try {
+            $table = $this->getElement('table');
             $rows = $this->tableAccessor->getRowsWithFields(
-                $this->getElement('table'),
+                $table,
                 ['item' => $productName]
             );
 
-            return 1 === count($rows);
+            foreach ($rows as $row) {
+                $field = $this->tableAccessor->getFieldFromRow($table, $row, 'item');
+                $name = $field->find('css', '.sylius-product-name');
```

Comment:
Shouldn't we assert that the field would exist?

Reply:
Not in this case, it would change the behaviour of this method.